platform/broadcom/saibcm-modules/include/kcom.h

```diff
@@ -470,8 +486,7 @@ typedef struct kcom_msg_filter_destroy_s {
  * Get list of currently defined packet filters.
  */
 #ifndef KCOM_FILTER_MAX
-/* SAI_FIXUP - Increased the filters to 1024 from 128 */
-#define KCOM_FILTER_MAX          1024
+#define KCOM_FILTER_MAX          128
```

Comment:
are you sure we are changing it back?

Reply:
This change came as part of BCM GPL SDK 6.5.16. I left it as such because there is the following compilation flag -DKCOM_FILTER_MAX=1024 which is present in the Makefile and will any ways override this.